pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -98919,7 +98919,6 @@ self: {
        description = "Console IRC client";
        license = stdenv.lib.licenses.isc;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
In general, this file shouldn't be edited by hand.  It is regenerated once a day or so, based on `configuration-hackage2nix.yaml `.

Can you remove the two edits here?